docs/src/index.md

```diff
@@ -268,6 +268,178 @@ df2 = @eachrow df begin
 end
 ```
 
+## Row-wise transformations with `@byrow`
+
+DataFrames provides the function-wrapper `ByRow`. `ByRow(f)(x, y)`
```

Comment:
I am not sure what style you follow in DataFramesMeta.jl, but in DataFrames.jl we always add .jl to package names

Reply:
added `.jl` but will make a point to do this before 1.0.